app/views/onboarding/Onboarding5.js

```diff
@@ -1,519 +0,0 @@
-import React, { Component } from 'react';
```

Comment:
If there's a way to force this to be marked at moved, that would be helpful. Not sure if git can do it though.

Reply:
probably, not sure if its worth the effort to figure out though.